js/src/lib/inferenceSnippets/inputs.ts

```diff
@@ -44,7 +46,7 @@ const inputsTextGeneration = () => `"Can you please let us know more details abo
 
 const inputsText2TextGeneration = () => `"The answer to the universe is"`;
 
-const inputsFillMask = (model: ModelData) => `"The answer to the universe is ${model.mask_token}."`;
+const inputsFillMask = (model: ModelPartial) => `"The answer to the universe is ${model.mask_token ?? "[unknown_mask_token]"}."`;
```

Comment:
on the hf-endpoints side, we sometimes don't have the mast token available right away.